src/mmrelay/plugin_loader.py

```
@@ -884,466 +884,697 @@ def _raise_install_error(pkg_name):
-                    if ref_type == "branch":
-                        logger.info(
-                            f"Cloned repository {repo_name} from {repo_url} at branch {ref_value}"
-                        )
-                    else:
-                        logger.info(
-                            f"Cloned repository {repo_name} from {repo_url} at tag {ref_value}"
-                        )
+                    logger.info(
+                        f"Cloned repository {repo_name} from {repo_url} at branch {other_default}"
+                    )
                     return True
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)